src/compose/ComposeBox.android.js

```diff
@@ -235,7 +235,7 @@ class ComposeBox extends PureComponent<Props, State> {
     const subject = topic !== editMessage.topic ? topic : undefined;
     if (content || subject) {
       updateMessage(auth, { content, subject }, editMessage.id).catch(error => {
-        showErrorAlert(error.message, 'Failed to edit message');
+        showErrorAlert(error.data.msg, 'Failed to edit message');
```

Comment:
You are making an assumption that the `error` object will always have a `data` property. If that assumption happens to be false, this will throw.

From the looks of the code, your assumption is currently true. Our API code should always throw through `makeApiError` and it seems that `removeEmptyValues` which is called from `updateMessage` is not going to throw...

Still, it might be a good idea to create a separate version of `showErrorAlert` that checks for `error.data`  before using the `msg` value. `showErrorFromException` or some similar name.

Reply:
@borisyankov Done 👍